src/themes/six/src/components/LoginForm/index.js

```diff
@@ -74,7 +74,7 @@ const LoginFormUI = (props) => {
   const [willVerifyOtpState, setWillVerifyOtpState] = useState(false)
   const [validPhoneFieldState, setValidPhoneField] = useState(false)
   const [otpState, setOtpState] = useState('')
-  const [otpLeftTime, _, resetOtpLeftTime] = useCountdownTimer(
+  const [otpLeftTime, resetOtpLeftTime] = useCountdownTimer(
```

Comment:
resetOtpLeftTime comes in the third parameter of the hook. Revert this change or send a initialOtpTime for each resetOtpLeftTime in the code

Reply:
Ok I will keep third parameter.